VotingApplication/VotingApplication.Web.Tests/Controllers/PollVoteControllerTests.cs

```diff
@@ -300,6 +306,45 @@ public void ClearingVoteGeneratesRemoveVoteMetric()
             _mockMetricHandler.Verify(m => m.HandleVoteDeletedEvent(existingVote, _mainUUID), Times.Once());
         }
 
+        [TestMethod]
+        [ExpectedHttpResponseException(HttpStatusCode.BadRequest)]
+        public void ElectionPoll_BallotHasVoted_BadRequest()
```

Comment:
Is "ElectionPoll_BallotHasVoted_ThrowsBadRequest()" a better name? Just from the name there's no indication that an exception is thrown.